src/njoy21/input/PLOTR/Card6/Ystep.hpp

```diff
@@ -0,0 +1,14 @@
+struct Ystep {
+  using Value_t = double;
+  static std::string name(){ return "ystep"; }
+  static std::string description(){
+    return
+      "The ystep argument specifies the spacing for tics on the y-axis.\n\n"
+      "The default behavior is automatic scaling.  The value is ignored if\n"
+      "log scaling is used.";
+  }
+  static Value_t defaultValue(){ return 1.0; }
```

Comment:
That does not seem to have anything to do with the value of ystep, rather the value of nz is dependent on what ystep has been set to.  Line 569 initializes the value to 0.  Line 579 seems to set ystep to 1, if a log scale is used for the y-axis, but only if the other values (yl and yh) are not defaulted.  nz seems to be used at line 580 to do the above check for all or none of the values to be defaulted.  In fact, it kind of looks like the check and set at line 579 should come after the error check on 580-581.

I can set it to 0.0 or 1.0 depending on the itype value, if you like.  It will get a bit confusing with the need to check on all or none of the values being default.

Reply:
You're right. I don't know what I was thinking there.